components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -580,33 +608,37 @@ def generic_start_scheduler(
         "-w", str(CONTAINER_CLP_HOME),
         "--name", container_name,
         "--log-driver", "local",
-        "-e", f"PYTHONPATH={clp_site_packages_dir}",
-        "-e", (
+        "-u", f"{os.getuid()}:{os.getgid()}",
+    ]
+    # fmt: on
+
+    necessary_env_vars = [
```

Comment:
Small nit: what if we turn this into a dictionary with EnvVarName:Value structure?